shelley/chain-and-ledger/formal-spec/utxo.tex

```diff
@@ -284,6 +284,8 @@ \subsection{UTxO Transitions}
       \\
       \forall (\_\mapsto (\_, c)) \in \txouts{tx}, c \geq 0
       \\
+      \fun{txsize}~{tx}\leq\fun{maxTxSize}~\var{pp}
```

Comment:
Maybe it'd be nice to state why did we decide to incorporate this constraint, as a way of documenting our design decisions :)